### PR TITLE
VJ/Add squad page Initial UI

### PR DESCRIPTION
**Pull request by vivekjain1397**
Add squad UI done: pick emoji, enter squad name, add squad name/emoji to squads page and events page.

### CODE REVIEW

**Comment by brianlevis** [comment]
can you delet `package_lock.json`

**Comment by abhishyantkhare** [comment]
@vivekjain1397 Can you squash some of these commits together? There's some commits where you comment code and then implement it in the next commit which makes it hard to follow. I think its cleaner too

File: imdown-mobile/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6721 @@
+{
```
Comment (abhishyantkhare) [inline imdown-mobile/package-lock.json:1]: delet this

File: imdown-mobile/squads/add_squad.tsx [hunk 2]
```diff
@@ -1,26 +1,72 @@
...
+    }
+
+    const renderEmojiPicker = () => {
+      return (
+        showEmojiPicker &&
+        <EmojiSelector
+          category={Categories.symbols}
+          onEmojiSelected={emoji => 
+            {
+              console.log(emoji)
```
Comment (abhishyantkhare) [inline imdown-mobile/squads/add_squad.tsx]: I feel like if there's more than 3 statements in the jsx-lambda then I would move this logic to its own function

File: imdown-mobile/squads/add_squad.tsx [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
+    }
 
     const renderEmoji = () => {
       return (
         <View >
-          <TouchableHighlight onPress={() => { setShowEmojiPicker(true) }} style={AddSquadStyles.add_squad_emoji}>
+          <TouchableOpacity onPress={() => { setShowEmojiPicker(true) }} style={AddSquadStyles.add_squad_emoji}>
             <Text style={AddSquadStyles.emoji}>
-             {`${emojiPicked ? squadEmoji : "TBD"}`}
+             {`${emojiPicked ? squadEmoji : "😎"}`}
```
Comment (abhishyantkhare) [inline imdown-mobile/squads/add_squad.tsx]: I know I did a bad job of this but we should move string literals to constants

File: imdown-mobile/squads/add_squad.tsx [hunk 4]
```diff
@@ -61,17 +64,32 @@ const AddSquadModal = (props: OwnProps) => {
...
 
+            {/* <View style={AddSquadStyles.addSquadButton}>
+              <TouchableOpacity onPress={addSquad} style={AddSquadStyles.add_squad_button}>
+                <Text style={AddSquadStyles.add_squad_text}>Done</Text>
+              </TouchableOpacity>
+            </View> */}
+
           </View>
 
+          {/* <View style={AddSquadStyles.addSquadButton}>
```
Comment (abhishyantkhare) [inline imdown-mobile/squads/add_squad.tsx]: no comments pls unless its explaining what the code is doing, just delet the code

File: imdown-mobile/squads/squads.tsx [hunk 5]
```diff
@@ -43,13 +43,13 @@ export type Squad ={
 // ]
 
 /* TODO:
-  - get squads to start at top
   - emojis propagate to squads scren */
```
Comment (abhishyantkhare) [inline imdown-mobile/squads/squads.tsx]: nit: typo scren

File: imdown-mobile/squads/squads.tsx [hunk 6]
```diff
@@ -43,13 +43,13 @@ export type Squad ={
...
 
 const Squads = (props) => {
 
   const [addSquadModalVisble, setAddSquadModalVisble] = useState(false)
   const [squads, setSquads] = useState(props.route.params.squads)
+  console.log(squads)
```
Comment (abhishyantkhare) [inline imdown-mobile/squads/squads.tsx]: no console log

File: imdown-mobile/package.json [hunk 7]
```diff
@@ -15,6 +15,7 @@
     "@react-navigation/native": "^5.0.5",
     "@react-navigation/stack": "^5.0.5",
     "emoji-picker-react": "^3.2.1",
+    "core-js": "^3",
```
Comment (abhishyantkhare) [inline imdown-mobile/package.json:18]: I would drop this commit in an interactive rebase and then pull changes from master using `git rebase origin master`
